membership_file/middleware.py

```diff
@@ -13,3 +14,14 @@ def process_request(self, request):
             request.member = Member.objects.filter(user__id=request.user.id).first()
         else:
             request.member = None
+
+    def process_exception(self, request, exception):
+        if isinstance(exception, UserIsNotCurrentMember):
+            return TemplateResponse(
+                request=request,
+                template='membership_file/no_member.html',
+                context={},
+                status=403,
+            )
+
+
```

Comment:
I know this is on me for not doing #13, but why the new lines?

Reply:
No conscious reason. Likely done so I had more visual clarity in my screen (blank space). I can remove them